components/server/src/jobs/installation-admin-cleanup.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { BUILTIN_INSTLLATION_ADMIN_USER_ID, UserDB } from "@gitpod/gitpod-db/lib";
```

Comment:
Nit
```suggestion
import { BUILTIN_INSTALLATION_ADMIN_USER_ID, UserDB } from "@gitpod/gitpod-db/lib";
```

Reply:
I've seen that. For the sake of a smaller diff, I've suppressed it

![image](https://github.com/gitpod-io/gitpod/assets/914497/2caba674-93bc-4481-a80d-d65a76da8c54)